src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -457,6 +457,27 @@ public virtual LocalRedirectResult LocalRedirectPermanentPreserveMethod(string l
         }
 
         /// <summary>
+        /// Redirects (<see cref="StatusCodes.Status302Found"/>) to action with the same name as current one.
+        /// The 'controller' and 'action' names are retrieved from the ambient values of the current request.
+        /// </summary>
+        /// <returns>The created <see cref="RedirectToActionResult"/> for the response.</returns>
+        /// <example>
+        /// A POST request to an action named "Product" creates a product and redirects to an action, also named
+        /// "Product", showing details of the created product.
+        ///
+        /// [HttpPost]
+        /// public IActionResult Product(Product product) {}
```

Comment:
This example doesn't show an example of actually using this API... is that deliberate? Also, I think examples probably need to have some kind of `<code>` tag or something, no? Otherwise I'm guessing it doesn't get formatted correctly?